exonum-java-binding/core/rust/ejb-app/start_cryptocurrency_node.sh

```diff
@@ -35,15 +35,20 @@ echo "CURRENT_DIR=${EJB_APP_DIR}"
 EJB_ROOT=$(realpath "../../..")
 echo "PROJ_ROOT=${EJB_ROOT}"
 
+# Find the artifact
+ARTIFACT_PATH="$(find ${EJB_ROOT} -type f -name exonum-java-binding-cryptocurrency-demo-*-artifact.jar)"
+echo "ARTIFACT_PATH=${ARTIFACT_PATH}"
+
+# Prepare the services configuration file
+SERVICES_CONFIG_FILE="ejb_app_services.toml"
+SERVICE_NAME="cryptocurrency-demo-service"
```

Comment:
Those names (as well as for `qa_service`) are required to be passed to Exonum via the `ServiceFactory` interface (and are used to get a list of services that are executed by node). I set it to the `JavaService.getName()` for consistency.